plumbing/object/commit_test.go

```diff
@@ -219,6 +220,7 @@ func (s *SuiteCommit) TestCommitEncodeDecodeIdempotent(c *C) {
 				plumbing.NewHash("f000000000000000000000000000000000000006"),
 				plumbing.NewHash("f000000000000000000000000000000000000007"),
 			},
+			Encoding: MessageEncoding("ISO-8859-1"),
```

Comment:
We need to verify the correct encoding was set. That can be done around line `234`.